### DISCUSSION TITLE
Cannot install Dev build - SEE ISSUE #876

### DISCUSSION THREAD

**Discussion by Delta18-Git** [File-New-Project/EarTrumpet#877] [Q&A] [answered]
Hi, I am not sure if this was supposed to go to the issues tab, but I made a discussion instead.
Whenever I try to install the dev build, App Installer says that is unable to parse the metadata through the website, and if I try to download the app installer from the link, it says that it took too long to load.

**Comment by riverar** [comment, marked as answer]
Yep, my fault. Dupe of issue #876. Standby.

**Reply by Delta18-Git** [reply]
Seems to be fixed.
